### ISSUE TITLE
JupyterLite 0.2.0( 0.1.2 and later) only works in Firefox

### ISSUE THREAD

**Issue by tanaga9** [jupyterlite/jupyterlite#1236]
## Description

JupyterLite 0.2.0 only works in Firefox

| browser | version | state | OS
| ---|---|---| ---|
| Firefox | 119.0 (64 bit) |  ✔ work properly (maybe) |  masOS Sonoma 14.0 |
| Google Chrome | 118.0.5993.117 (Official Build) (x86_64)  |  The page loads but hangs during processing. |  masOS Sonoma 14.0 |
| Safari | 17.0 (19616.1.27.211.1) | The page fails to load and the loading timeout is reached |  masOS Sonoma 14.0 |

I think ver 0.1.x worked on most browsers.

## Reproduce

I deployed a JupyterLite server on my local machine using Docker.

```
docker run --rm -it -p 8888:8888 python:3.10 bash -c '
pip install "jupyterlab>=4.0.0,<5" jupyterlite jupyterlite-core jupyterlite-pyodide-kernel jupyterlite-javascript-kernel
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```

I opened the some browsers and entered the URL http://127.0.0.1:8888/lab/index.html.

Google Chrome hangs
![スクリーンショット_sc_20231029_165838](https://github.com/jupyterlite/jupyterlite/assets/6501421/8261ba17-4300-4a3d-a0d8-886fda1b6fc9)

Safari does not load due to a timeout
![スクリーンショット_sc_20231029_165917](https://github.com/jupyterlite/jupyterlite/assets/6501421/b23f87ec-1e6f-4feb-90f4-aa77ffa3866f)
![スクリーンショット_sc_20231029_165927](https://github.com/jupyterlite/jupyterlite/assets/6501421/c532fe55-f5b7-4e59-9f2f-487103792025)

**Comment by tanaga9** [comment]
This problem seems to have been occurring for a while.

| JupyterLite version | Google Chrome	118.0.5993.117 (Official Build) (x86_64)| Firefox	119.0 (64 bit) |
|------| ----| ----|
| 0.2.0 | not working | ✔ work properly |
| 0.1.3 | not working | ✔ work properly |
| 0.1.2 | not working | ✔ work properly |
| 0.1.1 | ✔ work properly| ✔ work properly |
| 0.1.0 | ✔ work properly| ✔ work properly |

jupyterlite==0.1.3
```
docker run --rm -it -p 8888:8888 python:3.10 bash -c '
pip install "jupyterlab>=3.0.0,<4" jupyterlite==0.1.3 jupyterlite-core==0.1.3 jupyterlite-pyodide-kernel==0.1.3 jupyterlite-javascript-kernel==0.1.3
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```

jupyterlite==0.1.2
```
docker run --rm -it -p 8888:8888 python:3.10 bash -c '
pip install "jupyterlab>=3.0.0,<4" jupyterlite==0.1.2 jupyterlite-core==0.1.2 jupyterlite-pyodide-kernel==0.1.2 jupyterlite-javascript-kernel==0.1.2
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```

jupyterlite==0.1.1
```
docker run --rm -it -p 8888:8888 python:3.10 bash -c '
pip install "jupyterlab>=3.0.0,<4" jupyterlite==0.1.1 jupyterlite-core==0.1.1 jupyterlite-pyodide-kernel==0.1.1 jupyterlite-javascript-kernel==0.1.1
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```

jupyterlite==0.1.0
```
docker run --rm -it -p 8888:8888 python:3.10 bash -c '
pip install "jupyterlab>=3.0.0,<4" jupyterlite==0.1.0 jupyterlite-core==0.1.0 jupyterlite-pyodide-kernel==0.1.0 jupyterlite-javascript-kernel==0.1.0
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```

**Comment by tanaga9** [comment]
The following site seems to work on Google Chrome, but what is the difference?
https://jupyterlite.readthedocs.io/en/latest/_static/lab/index.html

**Comment by jtpio** [comment]
Thanks @tanaga9 for the detailed report.

This is odd, because deployments like the one you posted above, or https://github.com/jupyterlite/demo, have been working fine in Chrome for a while.

For example using Chromium 118:

![image](https://github.com/jupyterlite/jupyterlite/assets/591645/9a6e4fa0-d63a-4ba5-a08e-afda0a1527a0)

But it could be related to the browser cache, which is still an issue with JupyterLite at the moment (some issues opened on the repo).

Would you be able to try clearing the cache of the browsers you are using, just to make sure?

**Comment by tanaga9** [comment]
Hi @jtpio 

I tried specifying a port number that I had never used before, and it worked properly.

```
docker run --rm -it -p 10456:8888 python:3.10 bash -c '
pip install "jupyterlab>=3.0.0,<4" jupyterlite==0.1.3 jupyterlite-core==0.1.3 jupyterlite-pyodide-kernel==0.1.3 jupyterlite-javascript-kernel==0.1.3
jupyter lite build --output-dir _site && jupyter lite serve --port=8888 --ip=0.0.0.0 --output-dir _site
'
```
http://127.0.0.1:10456/lab/index.html.


I am using the [Clear Cache plugin in Google Chrome](https://chrome.google.com/webstore/detail/clear-cache/cppjkneekbjaeellbfkmgnhonkkjfpdn).

I had set the settings to clear the following three caches, but was it not enough?

- App Cache 
- Cache 
- Cache Storage 

Oops, I just realized that I accidentally set the Time Period to Last Hour, when I meant to set it to Everything.


It seems that the cache clearing was not thorough enough.
I apologize for the inconvenience.

I will monitor the situation for a while. If there are no problems, I will close this issue.

Thanks
